src/tools/pontoon/teams_projects.md

```diff
@@ -2,69 +2,74 @@
 
 <!-- toc -->
 
-The [Teams page](https://pontoon.mozilla.org/teams/) lists all locales that are enabled in Pontoon. From here you can access a specific [Team page](#team-page), where you’ll get a list of all projects enabled for the requested locale. Selecting a project from the *Team page* will take you to what is called the [Localization page](#localization-page).
+The [Teams page](https://pontoon.mozilla.org/teams/) lists all locales that are enabled in Pontoon. From here specific [Team pages](#team-page) can be accessed, which contains a list of all projects enabled for the requested locale. Selecting a project from the *Team page* leads to what is called the [Localization page](#localization-page).
```

Comment:
```suggestion
The Teams page, accessible using the `/teams` URL (e.g. [pontoon.mozilla.org/teams](https://pontoon.mozilla.org/teams)), lists all locales that are enabled in Pontoon. From here it’s possible to access a specific [Team page](#team-page), which contains a list of all projects enabled for the requested locale. Selecting a project from the *Team page* leads to what is called the [Localization page](#localization-page).
```

We need to use the same strategy we used in other docs, where pontoon.mozilla.org is an example, and this is a generic documentation.